core/templates/dev/head/pages/exploration_editor/statistics_tab/EarlyQuitIssueDirective.js

```diff
@@ -0,0 +1,157 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
```

Comment:
As discussed, split up the dependencies of this PR so we can review/merge them each individually.